### PR TITLE
[APB-4345][PT]: Add Create Relationship for CGT on Accept

### PR DESCRIPTION
**Pull request by JTY009**

### CODE REVIEW

File: app/uk/gov/hmrc/agentclientauthorisation/connectors/RelationshipsConnector.scala [hunk 1]
```diff
@@ -66,6 +66,12 @@ class RelationshipsConnector @Inject()(
       http.PUT[String, HttpResponse](trustRelationshipUrl(invitation).toString, "") map (_ => Unit)
     }
 
+  def createCapitalGainsRelationship(
+    invitation: Invitation)(implicit hc: HeaderCarrier, ec: ExecutionContext): Future[Unit] =
+    monitor(s"ConsumedAPI-AgentClientRelationships-relationships-CapitalGains-PUT") {
+      http.PUT[String, HttpResponse](cgtRelationshipUrl(invitation).toString, "") map (_ => Unit)
```
Comment (sureshhmrc) [inline app/uk/gov/hmrc/agentclientauthorisation/connectors/RelationshipsConnector.scala]: shouldn't this be `()`  in place of `Unit`?

Comment (JTY009) [reply app/uk/gov/hmrc/agentclientauthorisation/connectors/RelationshipsConnector.scala]: ok

File: app/uk/gov/hmrc/agentclientauthorisation/connectors/AuthActions.scala [hunk 2]
```diff
@@ -149,6 +149,7 @@ class AuthActions @Inject()(metrics: Metrics, val authConnector: AuthConnector)
               }
             case Left(error) => Future successful error
           }
+        case _ ~ _ => Future successful GenericForbidden
```
[the remark this answers is not in the document]
Comment (JTY009) [reply app/uk/gov/hmrc/agentclientauthorisation/connectors/AuthActions.scala]: yes was overzealous

File: app/uk/gov/hmrc/agentclientauthorisation/service/InvitationsService.scala [hunk 3]
```diff
@@ -134,7 +134,7 @@ class InvitationsService @Inject()(
           result <- changeInvitationStatusAndRecover
           _ <- result match {
                 case Right(invite) => emailService.sendAcceptedEmail(invite)
-                case Left(_)       => Future successful ()
+                case Left(_)       => Future successful Unit
```
[the remark this answers is not in the document]
Comment (JTY009) [reply app/uk/gov/hmrc/agentclientauthorisation/service/InvitationsService.scala]: can

**Pull request merged**